readme.md

```diff
@@ -385,90 +507,53 @@ interface SubmitTerms {
      *  @link https://kiltprotocol.github.io/sdk-js/interfaces/types_src.ICType.html */
     cTypes: ICType[]
```

Comment:
This should be the following:

`
```suggestion
    cTypes: Array<{
        cType: PartialClaim
    }>;
```

Reply:
However, this should be adapted to the changes made by @rflechtner

Reply:
Could you elaborate a bit? Why would you change it like that, which changes done by Raphael where?